readme.md

```diff
@@ -82,4 +86,4 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 ### Blogs
 
 - [Official Selenium Blog](http://seleniumhq.wordpress.com/) - The official blog by SeleniumHQ.
-- [Elemental Selenium](http://elementalselenium.com/) - A free, once-weekly e-mail on how to use Selenium like a Pro.
\ No newline at end of file
+- [Elemental Selenium](http://elementalselenium.com/) - A free, once-weekly e-mail on how to use Selenium like a Pro.
```

Comment:
if this is newline at EOF is a problem let me know :+1: 

the built-in github editor doesn't do a great job abiding by the contribution guidelines itself.